frontend/src/Troops/index.jsx

```diff
@@ -22,36 +22,36 @@ function Troops() {
 
   const { listOfTroops, isLoading, error } = useSelector((state) => state.troops);
 
-  const [troop] = listOfTroops;
-
   if (error) {
     return <ErrorPopup message={`Oops, ${error.message}`} />;
   }
 
-  if (isLoading || !troop) {
+  if (isLoading || Object.keys(listOfTroops.length === 0)) {
     return <ActivityIndicator size="large" color={Colors.tealColor} />;
   }
 
   const {
     attack, defence, hp: sustenance, countByLevel,
-  } = troop;
+  } = listOfTroops;
```

Comment:
it's rather details of troops and not an actual list, isn't it? as I see it's not the troops but the information about them so the name is confusing